python/GafferUI/SpreadsheetUI.py

```diff
@@ -1588,15 +1617,28 @@ def __nodeEditorToolMenu( nodeEditor, node, menuDefinition ) :
 	assert( not ( selectorValue and selectorContextVariablePlug ) )
 
 	menuDefinition.append( "/SpreadsheetDivider", { "divider" : True } )
+
+	itemsActive = (
+		not nodeEditor.getReadOnly()
+		and not Gaffer.MetadataAlgo.readOnly( node )
+		and not Gaffer.MetadataAlgo.readOnly( activeRowNamesConnection )
+	)
+
 	menuDefinition.append(
 		"/Create Spreadsheet...",
 		{
 			"command" : functools.partial( __createSpreadsheetForNode, node, activeRowNamesConnection, selectorContextVariablePlug, selectorValue ),
-			"active" : (
-				not nodeEditor.getReadOnly()
-				and not Gaffer.MetadataAlgo.readOnly( node )
-				and not Gaffer.MetadataAlgo.readOnly( activeRowNamesConnection )
-			)
+			"active" : itemsActive
+		}
+	)
+
+	driveCommand = functools.partial( __drivePlugFromRowNames, activeRowNamesConnection, selectorContextVariablePlug, selectorValue )
+	menuDefinition.append(
+		"/Drive from Spreadsheet",
```

Comment:
Maybe "Connect to Spreadsheet"? We do modify the spreadsheet in this interaction to, so it's a kindof bidirectional thing. I really wanted to implement it so that `selectorContextVariablePlug` drove `selector` dynamically by being connected to it, but was thwarted by the need to add surrounding "${}".

Reply:
Renamed in https://github.com/GafferHQ/gaffer/pull/3560/commits/f860150d1bd80cfa4e300f47fb397198f7844d78